sdk/src/utils.ts

```diff
@@ -175,3 +180,46 @@ export function getMerkleProof(block: Block, txHash: string, forWitness?: boolea
         root: merkleAndRoot.root.toString("hex"),
     };
 }
+
+export function estimateTxFee(feeRate: number, numInputs: number = 1) {
```

Comment:
should document the unit of the feeRate (I've think I've seen 3 different units being used in other code before)